src/main/java/nextstep/subway/application/StationService.java

```diff
@@ -22,14 +22,14 @@ public StationService(StationRepository stationRepository) {
     @Transactional
     public StationResponse saveStation(StationRequest stationRequest) {
         Station persistStation = stationRepository.save(stationRequest.toStation());
-        return StationResponse.of(persistStation);
+        return StationResponse.from(persistStation);
     }
 
     public List<StationResponse> findAllStations() {
         List<Station> stations = stationRepository.findAll();
 
         return stations.stream()
-                .map(station -> StationResponse.of(station))
+                .map(station -> StationResponse.from(station))
```

Comment:
```suggestion
                .map(StationResponse::from)
```
메서드 레퍼런스도 활용해보면 좋을 것 같아요 :)